src/cdk/v2/destinations/zoho/config.js

```diff
@@ -0,0 +1,66 @@
+const SUPPORTED_EVENT_TYPE = ['record'];
+const ACTION_TYPES = ['insert', 'delete'];
```

Comment:
why `update` is missing ?

Reply:
Removing this. Not using it at all. All insert, update, delete actions are supported